README.md

```diff
@@ -1,187 +1,38 @@
 # Warpgrapher
+[![Build Status](https://github.com/warpforge/warpgrapher/workflows/Test/badge.svg)](https://github.com/warpforge/warpgrapher/actions?query=workflow%3A%22Test%22+branch%3Amaster)
+[![GitHub Release](https://img.shields.io/github/release/warpforge/warpgrapher.svg?style=flat)](https://github.com/warpforge/warpgrapher/releases/tag/v0.2.0)
+[![Crates.io](https://img.shields.io/crates/v/warpgrapher.svg)](https://crates.io/crates/warpgrapher)
 
-Warpgrapher makes it painless to create web services with graph-based data
-models. Describe the data model for which you want to run a web service.
-Wargrapher automatically generates a GraphQL schema from the data model, as well
-as a set of resolvers for basic reate, read, update, and delete (CRUD)
-operations on that data.
+#### Framework for creating data model-driven GraphQL API services
 
-If you need more more sophisticated, custom queries and endpoints, you can
-supply your own custom resolvers. Warpgrapher will automatically generate the
-GraphQL configuration and invoke your custom resolvers when appropriate.
+As a developer you can focus on defining your applications data model and warpgrapher takes care of generating a graph-based API for interacting with that model. In addition to generating CRUD APIs for interacting with the model, warpgrapher provides a set of advanced features to customize and extend your service. 
```

Comment:
The fragment "your applications," should be "your application's."  Also, "data model and warpgrapher" should be "data model, and warpgrapher."  Should warpgrapher be capitalized when used, even mid-sentence?  Is is a proper noun, as a named project, so I would think yes, but as long as we're consistent, I'm okay with either.

Reply:
Fixed (using uppercase).